qubesbuilder/helpers.py

```diff
@@ -0,0 +1,71 @@
+import sys
+from pathlib import Path
+
+from qubesbuilder.executors import ExecutorError
+from qubesbuilder.executors.container import ContainerExecutor
+from qubesbuilder.executors.local import LocalExecutor
+from qubesbuilder.executors.qubes import QubesExecutor
+from qubesbuilder.plugins import PluginError
+
+
+def get_executor(executor_type, executor_options):
+    if executor_type in ("podman", "docker"):
+        executor = ContainerExecutor(executor_type, **executor_options)
+    elif executor_type == "local":
+        executor = LocalExecutor(**executor_options)  # type: ignore
+    elif executor_type == "qubes":
+        executor = QubesExecutor(**executor_options)  # type: ignore
+    else:
+        raise ExecutorError("Cannot determine which executor to use.")
+    return executor
+
+
+def _load_modules(path):
+    loaded_modules = []
+    modules = Path(path).iterdir()
+    module_pattern = "qubesbuilder.plugins.%s"
+
+    for directory in modules:
+        if not (directory / "__init__.py").exists():
+            continue
+        mod_name = directory.name
+        if mod_name == "plugins":
+            continue
+        try:
+            module = sys.modules.get(module_pattern % mod_name)
+            if not module:
+                __import__(module_pattern % mod_name)
+                module = sys.modules[module_pattern % mod_name]
```

Comment:
This won't load plugins from `path`, but from standard python locations, and look for `qubesbuilder.plugin` specifically. In other words: this won't work for plugins outside of the builder repo.